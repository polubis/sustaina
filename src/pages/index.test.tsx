import React from 'react';
import HomePage from './index';
import { render } from '@testing-library/react';

jest.mock(`gatsby`, () => ({
  graphql: () => ({
    site: {
      siteMetadata: {
        title: `Some title`,
        siteUrl: `http://localhost:3000`,
      },
    },
  }),
  useStaticQuery: (result: unknown) => result,
}));

describe(`Home page works when: `, () => {
  it(`displays content`, () => {
    const { asFragment } = render(<HomePage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
