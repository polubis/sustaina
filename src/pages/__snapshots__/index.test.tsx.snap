// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page works when:  displays content 1`] = `
<DocumentFragment>
  <main>
    <h1
      class="text-3xl top-5 font-bold underline"
    >
      Hello!
    </h1>
    <span
      class="inline-flex items-center rounded-md bg-gray-50 px-2 py-1 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10"
    >
      Badge
    </span>
    <span
      class="dark:text-white text-black"
    >
      Theme test
    </span>
    <h1
      class="text-3xl top-5 font-bold underline"
    >
      This is home page. Zustand works! The blog title is:Some title
    </h1>
  </main>
</DocumentFragment>
`;
